src/modules/fancyzones/editor/FancyZonesEditor/Properties/Resources.resx

```diff
@@ -405,7 +405,7 @@
     <value>custom layout was created successfully.</value>
   </data>
   <data name="OpenSettings" xml:space="preserve">
-    <value>Open settings</value>
+    <value>Go to settings</value>

```

Comment:
this may be a dead string

Reply:
Then it should be removed. Which has nothing to do with pr. So what do you want me to do about this?

Reply:
If you're going in and altering code, part of that is testing and seeing what got adjusted.

Going in, it was commented out.

![image](https://github.com/microsoft/PowerToys/assets/1462282/01d72249-0871-4d9e-9806-e6a3cef4fbe9)

Reply:
Then I assume it's alright (?)